lineman/app/app.coffee

```diff
@@ -17,10 +17,14 @@ angular.module('loomioApp', ['ngRoute',
 angular.module('loomioApp').config ($translateProvider) ->
   $translateProvider.
     useUrlLoader('/api/v1/translations/en').
-    preferredLanguage('en')
+    preferredLanguage('pt-BR')
```

Comment:
I'd like to know how we get `Records.users.find(window.Loomio.currentUserId).selectedLocale` or `UserAuthService.currentUser.selectedLocale` in here